bot/exts/info/resources.py

```diff
@@ -0,0 +1,74 @@
+import re
+from typing import Optional
+from urllib.parse import quote
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+REGEX_CONSECUTIVE_NON_LETTERS = r"[^A-Za-z0-9]+"
+RESOURCE_URL = "https://www.pythondiscord.com/resources/"
+
+
+def _to_kebabcase(resource_topic: str) -> str:
+    """
+    Convert any string to kebab-case.
+
+    For example, convert
+    "__Favorite FROOT¤#/$?is----LeMON???" to
+    "favorite-froot-is-lemon"
+    Code adopted from:
```

Comment:
Minor, but makes things a little clearer IMO:
```suggestion
    "favorite-froot-is-lemon"
    
    Code adopted from:
```